README.md

```diff
@@ -1,3 +1,16 @@
 
 # Feverbase
 An open platform to accelerate any and all research related to clinical trials (including but not limited to the efficacy of repurposed drugs) for fighting COVID-19.
+## Want to Help?
```

Comment:
Would recommend a couple of sections (and nix want to help title)

Dependencies (and how to install them):
- Meili, Mongo, etc.
- Local dependencies with pip -r req.txt

Running the app
- serve.py
- fetch.py (perhaps even let people run the scripts directly)

Contributing
- idk if we need a section on this people know it's open source and can contribute/issues

License
This is important. We're MIT licensed I think.